Virtual agent

haku-c opened the pull request:
Adds an option to render replays without the sphere from the collected data set.

Commit:
Initial modifications
- Path changes in config
- initial sbatch script for rendering

haku-c committed:
Changes to run replay_episodes on cluster

haku-c committed:
Add config option for invisible agent
- invisible agent can render existing environments with smaller sphere
- add xml files for invisible agent assets and environments

haku-c committed:
Remove small sphere from invisible agent

haku-c committed:
Undo unncessary rgba changes

haku-c committed:
Changing config file directories to portal

haku-c committed:
Render cabinet movement
- hard code numpy array to open and close cabinet

haku-c committed:
Helper functions to perform actions from xskills
- Modify speed of each action
- Change orders

haku-c committed:
Add pause duration between actions

haku-c committed:
Tasks with sub goals

haku-c committed:
Add support for different subgoals in a task
- vary speed of subgoals
- pauses between subgoals
- use a separate config file for generate new renders

haku-c committed:
Add feature for completion percentage for sliders
- Tested on microwave and slide hinge

haku-c committed:
Set goals for max opening of certain tasks

haku-c committed:
Add function interpolation for rendering

haku-c committed:
Redo create pos arguments by using dictionaries

haku-c committed:
Simultaneous action implementation

haku-c committed:
Add docstrings to functions

haku-c committed:
Reformat documentation comments

haku-c commented:
Add options for modifying action duration, completion percentage for opening tasks, different easing functions for motion of tasks, and simultaneous tasks.

kushal2000 commented on config/simulation/replay_kitchen.yaml:
Don't think it is correct to change the video_path directory here to "renders" since it is used downstream by the policy training code. Should change this to "datasets" instead

haku-c replied:
Yeah I made a separate config file to control the invisible agent renders. As such, I no longer use this replay_kitchen config file but I forgot to switch it back.

kushal2000 commented on job-generate-render:
Should not commit this

kushal2000 commented on xskill/env/kitchen/relay_policy_learning/adept_envs/adept_envs/franka/assets/franka_kitchen_jntpos_act_ab_none.xml:
what is this file?

haku-c replied:
This file is a modified version of the frank kitchen environment used to remove the sphere object and other embodiment objects.

kushal2000 commented on xskill/env/kitchen/relay_policy_learning/adept_envs/adept_envs/franka/kitchen_multitask_v0.py:
seems like cosmetic changes only

haku-c replied:
Yeah I use an autoformatter so most likely that's what happened.

kushal2000 commented on job-render:
Should not commit this

kushal2000 commented on job-render.sb:
Should not commit this

haku-c committed:
Cleanup for merge into main
- Remove autoformatter change to kitchen_multitask_v0
- Remove job scripts from repo
- Revert generate_kitchen.yaml paths

haku-c committed:
Revert paths in replay_kitchen.yaml

haku-c committed:
default embodiment for invisible renders is none

haku-c committed:
Remove unused python script

haku-c committed:
Revert kitchen_multitask_v0.py to work with replay episodes
